filament/src/details/Camera.cpp

```diff
@@ -165,7 +165,8 @@ void UTILS_NOINLINE FCamera::setProjection(Camera::Projection projection,
 }
 
 math::mat4 FCamera::getProjectionMatrix(uint8_t eye) const noexcept {
-    assert_invariant(eye < CONFIG_STEREOSCOPIC_EYES);
+    UTILS_UNUSED_IN_RELEASE const Engine::Config& config = mEngine.getConfig();
```

Comment:
I'm starting to think that `Camera` should be created with an eye count, 1 by default. It would be cleaner if Camera knew how many eyes it has, instead of having to ask the engine. Also we could validate things better, for example that the camera matches the view's stereo mode.

Reply:
Yeah I think I agree. Let's do this in a separate PR, since that will be another API change?

Reply:
sgtm